src/expressionDescriptor.ts

```diff
@@ -461,7 +461,20 @@ export class ExpressionDescriptor {
       description = allDescription;
     } else if (!StringUtilities.containsAny(expression, ["/", "-", ","])) {
       description = StringUtilities.format(getDescriptionFormat(expression), getSingleItemDescription(expression));
-    } else if (expression.indexOf("/") > -1) {
+    } else if (expression.indexOf("/") > -1 && expression.indexOf(",") > -1) {
+      let segments: string[] = expression.split(",");
+      let segmentDescriptions: string[] = [];
+      for (let i = 0; i < segments.length; i++) {
+        segmentDescriptions.push(this.getSegmentDescription(segments[i],
+          allDescription,
+          getSingleItemDescription,
+          getIntervalDescriptionFormat,
+          getBetweenDescriptionFormat,
+          getDescriptionFormat
+        ));
+      }
+      description = segmentDescriptions.join(", ");
```

Comment:
Although I like your succinct approach with `.join`, the handling for `expression.indexOf(",") > -1 && expression.indexOf("/") == -1` starting on line 523 handles formatting of multiple values by prefixing last with ", and" and a few other things.  To keep it consistent I'm hoping we can use a similar approach.

Here is a suggested change:

```suggestion
      let segments: string[] = expression.split(",");
      let descriptionContent: string = "";
      for (let i = 0; i < segments.length; i++) {
        if (i > 0) {
          descriptionContent += ",";

          if (i < segments.length - 1) {
            descriptionContent += " ";
          }
        }

        if (i > 0 && segments.length > 1 && (i == segments.length - 1 || segments.length == 2)) {
          descriptionContent += `${this.i18n.spaceAnd()} `;
        }

        descriptionContent += this.getSegmentDescription(segments[i],
          allDescription,
          getSingleItemDescription,
          getIntervalDescriptionFormat,
          getBetweenDescriptionFormat,
          getDescriptionFormat
        );
      }

      description = descriptionContent;
```